File: PrimeCheck/primecheck.java
class primecheck{
    public static void main(String[] args) {
        System.out.println(isPrime(9));
        
    }
    public static boolean isPrime(int n) {
        boolean check = true;
        for (int i = 2; i <= (int) Math.sqrt(n); i++) {
            if(n % i == 0){
                check = false;
                break;
            }
        }
        return check;
    }
}

File: PrimeCheck/primecheck.py
from math import sqrt
from itertools import count,islice
import time

def isPrime(n):
    check = True
    for x in range(2,int(sqrt(n)) + 1):
        if n%x == 0:
            check = False
            break
    return check

def isPrime2(n):
    if n < 2: return False
    return all(n%i for i in islice(count(2), int(sqrt(n)-1)))


